package.json

```diff
@@ -255,7 +259,7 @@
     "ts-node": "^10.9.2",
     "tsc-alias": "^1.8.8",
     "tsconfig-paths-webpack-plugin": "^4.2.0",
-    "typescript": "5.5.3"
+    "typescript": "5.7.3"
```

Comment:
There's a resolution for typescript, so I don't think this does anything?  Also, I think typescript is a dep of all packages – might be best to keep them locked to same version?